packages/auth/amplify_auth_cognito/example/android/app/src/main/kotlin/com/amazonaws/amplify/example/MainActivity.kt

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
Are we enforcing adding licenses to flutter generated files in sample apps?

Reply:
We're not enforcing headers yet, but I do think it's easier than listing out each one of the generated files in license.sh! xD